docs/features/web-api-template.md

```diff
@@ -38,6 +38,7 @@ And additional features available with options:
   * `Certificate`: adds [client certificate authentication](https://webapi.arcus-azure.net/features/security/auth/certificate) mechanism to the API project
   * `None`: no authentication configured on the API project.
 * `-ia|--include-appsettings` (default `false`): includes a `appsettings.json` file to the web API project.
+* `-c|--correlation` (default `false`): adds correlation on transaction and operation level.
```

Comment:
We should turn this around and opt-out of correlation since we want to provide this out-of-the-box